README.md

```diff
@@ -22,58 +22,62 @@
 
 </h1>
 
-# pixi: Package management made easy
+# pixi: Package Management Made Easy
 
-`pixi` is a cross-platform, multi-language package manager and workflow tool
-built on the shoulders of the conda ecosystem.
+- [Overview](#overview)
+- [Highlights](#highlights)
+- [Getting Started](#getting-started)
+- [Status](#status)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [Built Using Pixi](#built-using-pixi)
```

Comment:
Im not a fan of the TOC , I'd like to start with the overview, we have another TOC below that as well.

Reply:
Removed it.